metrics/util/additionalmetrics.go

```diff
@@ -0,0 +1,21 @@
+package util
```

Comment:
I think it's this is wrong location for `AdditionalMetrics` that should be in `rest` package, similar to `serverExtraLogger`